tests/src/JIT/HardwareIntrinsics/X86/Regression/GitHub_18041/GitHub_18041.cs

```diff
@@ -0,0 +1,38 @@
+using System;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Runtime.Intrinsics.X86;
+using System.Runtime.Intrinsics;
+
+namespace GitHub_17435
```

Comment:
I don't think we need this test. This is not a regression and was simply not caught because the original test (which was fixed below) was calling the wrong method.

Reply:
Second

Reply:
Will remove